src/__tests__/element-queries.js

```diff
@@ -375,6 +383,22 @@ test('using jest helpers to check element class names', () => {
   ).toThrowError()
 })
 
+test('using jest helpers to check element role', () => {
+  const {getByRole} = render(`
+    <div role="dialog">
+      <span>Contents</span>
+    </div>
+  `)
+
+  expect(getByRole('dialog')).toBeTruthy()
+  expect(getByRole('dialog')).toHaveTextContent('Contents')
+
+  expect(() => expect(getByRole('dialog2')).not.toBeTruthy()).toThrowError()
```

Comment:
These other assertions are unnecessary because they're basically how jest-dom works and we don't need to worry about testing jest-dom :)

Reply:
Removed assertions

I basically copied from other tests so I wasn't sure how bullet proof you wanted it.